corehq/apps/app_manager/views.py

```diff
@@ -348,7 +348,8 @@ def copy_app(request, domain):
 def app_from_template(request, domain, slug):
     _clear_app_cache(request, domain)
     template = load_app_template(slug)
-    app = import_app_util(template, domain, {
+    source = template.export_json(dump_json=False)
```

Comment:
I wonder if this shouldn't actually happen in the import_app function to be even more defensive?